e2e/integration/subjects_page.spec.ts

```diff
@@ -11,10 +11,10 @@ describe('Subjects page', () => {
     cy.fixCypressSpec('/e2e/integration/subjects_page.spec.ts');
     cy.visit('/?disableSSR=true');
 
-    cy.gqlIntercept({
+    /*cy.gqlIntercept({
       alias: 'alerts',
       operations: ['alerts'],
-    });
+    });*/
```

Comment:
Kommenterte ut denne fordi den feila lokalt.

Reply:
GQL har nok bestemt seg for å merge denne med et annet query som gjøres samtidig. Kan fikses opp i ved å sjekke at `operations` i testen matcher med POST-kallene til GQL.

Reply:
Skjønte ikkje heilt kva du meinte her.

Reply:
Tror ikke den trenger å være kommentert ut. Funker fint hos meg!

Reply:
Det feiler lokalt men funker i actions. Kan la det være inne.